docs/Contributing.asciidoc

```diff
@@ -57,6 +57,7 @@ before every commit call:
 [source,sh]
 ----
 make tidy
+make js-tidy
```

Comment:
We don't want everyone to need to call *two* commands. Why not just call `js-tidy` as part of tidy then? Possibly rename the existing "tidy" to "tidy-perl" and have an aggregate make target

Reply:
Fixed.